index.js

```diff
@@ -62,9 +63,16 @@ function choo (opts) {
   // (str?, obj?) -> DOMNode
   function start () {
     _store.model(createLocationModel(opts))
-    const createSend = _store.start(opts)
+    const createSend = _store.start(xtend(opts, {
+      subscriptions: false,
+      noSubscriptions: true
```

Comment:
oops, yeah that's an artifact: `noSubscriptions` was deprecated & should not be used